tools/codegen/base/wash.json

```diff
@@ -0,0 +1,13 @@
+{
+  "repository": "https://github.com/wasmCloud/wasmCloud",
+  "tag_prefix": "wash-cli-v",
```

Comment:
Are you sure this is correct?

I don't think wasmCloud has a tag prefix?

https://github.com/wasmCloud/wasmCloud/releases/download/v1.4.2/wasmcloud-aarch64-apple-darwin

Reply:
I guess this pr is about wash (wash-cli), not wasmcloud.

Reply:
The intent was indeed to add `wash` (the "wasmCloud shell" CLI tool), because it has plenty of uses in GitHub Actions, not wasmCloud itself 🙂

Reply:
Thanks for clarifying!